src/ResultsTableHOC.js

```diff
@@ -23,10 +23,10 @@ let getAddOpts = ({ include, schema, field, setInclude, setIsAdding }) => {
   let addableFields = _.difference(schemaFields(schema), include)
   return [
     ..._.map(
-      f => ({
+      (f) => ({
```

Comment:
@timothyhoover can you update the linter to always remove outer parens on one-argument arrow functions rather than always adding them?

Reply:
I'll get it actually installed on this project in the next couple of days